documents/postman/staging-simplified.postman_environment.json

```diff
@@ -0,0 +1,71 @@
+{
+	"id": "1acffa98-51c5-4ba7-87fc-9f3d460c38ba",
+	"name": "development Copy",
+	"values": [
+		{
+			"key": "email-from",
+			"value": "solutions",
+			"enabled": true
+		},
+		{
+			"key": "email-template-id",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "notification-api-url",
```

Comment:
You're missing a url variable. It should have both notification-api-url and notification-api-url-private (which works on the VA network).